includes/SpecialVerifiedExport.php

```diff
@@ -0,0 +1,535 @@
+<?php
+
+namespace DataAccounting;
+
+use HTMLForm;
+use HTMLTextAreaField;
+use MediaWiki\Logger\LoggerFactory;
+use MediaWiki\MediaWikiServices;
+use SpecialPage;
+use Title;
+use WikiExporter;
+use Wikimedia\Rdbms\ILoadBalancer;
+
+class SpecialVerifiedExport extends SpecialPage {
```

Comment:
Need to document the list of DataAccounting-specific modifications, and also in the lines of the modifications, say that it is "DataAccounting modification" so that it is easy to grep.

Reply:
I'd like to throw out everything that is not needed (see question https://github.com/inblockio/DataAccounting/issues/146#issuecomment-969039157) and then clean up the remainder. Like that it won't be practical to document inline what changed. It is already tricky since we changed the order of some code.

Reply:
Some class level doc would still be possible, but perhaps would be best focused on explaining on how the page differs from the users perspective? Again I'm inclined to wait a bit till there is a better idea of what the page should look like.

Reply:
The only blocker for this PR to be merged is to document the behavioral difference from `Special:Export`. I think it is fine to document just the diff of current behavior. Future deviations can be documented later. Other than https://github.com/inblockio/DataAccounting/blob/b8181294f2ec3bc516ce7096d42015d824ccfe30/includes/SpecialVerifiedExport.php#L44-L45, what else needs to be documented?

Reply:
These are the behavior changes

![image](https://user-images.githubusercontent.com/146040/142771659-2a84b0f5-6ed9-47b5-8ce7-611c467e33ed.png)